src/chassis.hpp

```diff
@@ -136,7 +260,16 @@ class Chassis final :
         void supplementByStaticFields(
             const DBusInstancePtr& instance) const override
         {
-            instance->supplementOrUpdate(fieldName, std::string("MainChassis"));
+            setFieldName(instance, "MainChassis");
+        }        
```

Comment:
trailing spaces

Reply:
Looks like this is artifact of my intermediate implementation, I'll remove this.